apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -426,16 +426,41 @@ const SwapForm = () => {
         },
       });
     } else {
-      setDrawer(
-        ExchangeDrawer,
-        {
-          swapTransaction,
-          exchangeRate,
-        },
-        {
-          preventBackdropClick: true,
-        },
-      );
+      const swapWebApp = !!process.env.SWAP_WEB_APP;
+      if (swapWebApp) {
```

Comment:
To ease the readability of the code, you can extract this part of the code into a function and call it into this "if" statement.